tests/units/test_change_password.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FIWARE project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+
+__author__ = 'gjp'
+
+from os import environ
+from unittest import TestCase
+from skuld import change_password
+from mock import patch
+from test_openstackmap import MySessionMock
+
+
+class TestChangePassword(TestCase):
+    mock_session2 = MySessionMock()
+
+    def setUp(self):
+        self.OS_AUTH_URL = 'http://cloud.lab.fi-ware.org:4731/v2.0'
+        self.OS_USERNAME = 'user'
+        self.OS_PASSWORD = 'password'
+        self.OS_TENANT_NAME = 'user cloud'
+        self.OS_TENANT_ID = '00000000000000000000000000000001'
```

Comment:
if it is no sense to put into a configuration variable put it as a common variable either in test_change_password.py or in the other one (i prefer the other due to it is where to make the comparison).